### ISSUE TITLE
[Documentation] Error in the Constrained JSON sample

### ISSUE THREAD

**Issue by dilinisg** [ballerinalang/ballerina#3426]
Errors shown when the Constrained JSON sample is copied from the ballerina by example site.

Reason is due to having multiple line doc descriptions as follows.

```
@doc:Description {
        value: "Resource to get employee for the person by index which" +
        " can be invoked with /people/employee?index={index}"
    }
```

**Comment by samgnaniah** [comment]
Assign to me.

**Comment by samgnaniah** [comment]
Cannot reproduce this. Please check: https://github.com/ballerinalang/ballerina/tree/master/samples/ballerina-by-example/examples/constrained-json

**Comment by anupama-pathirage** [comment]
This issue is fixed with 0.94.0 release

https://ballerinalang.org/docs/by-example/constrained-json
